Dockerfile

```diff
@@ -1,15 +1,31 @@
-FROM debian:latest AS builder
-LABEL maintainer="roman.atachiants@gmail.com"
+FROM golang:1.14 AS builder
```

Comment:
Let's change this to `golang:1.16` or even `golang:1` (which should be good given Go 1.0 compatibility promise)